bugtool/cmd/root.go

```diff
@@ -81,10 +81,10 @@ func init() {
 	BugtoolRootCmd.Flags().BoolVar(&getPProf, "get-pprof", false, "When set, only gets the pprof traces from the cilium-agent binary")
 	BugtoolRootCmd.Flags().BoolVar(&envoyDump, "envoy-dump", true, "When set, dump envoy configuration from unix socket")
 	BugtoolRootCmd.Flags().IntVar(&pprofPort,
-		"pprof-port", defaults.GopsPortAgent,
+		"pprof-port", defaults.PprofPortAgent,
 		fmt.Sprintf(
-			"Pprof port to connect to. Known Cilium component ports are agent:%d, operator:%d, apiserver:%d",
-			defaults.GopsPortAgent, defaults.GopsPortOperator, defaults.GopsPortApiserver,
```

Comment:
i guess we don't need `apiserver` anymore?

Reply:
That's true. I removed it.

Reply:
alright i trust you

Reply:
The apiserver refers to the clustermesh-apiserver? If so, why don't we need it anymore?

Reply:
Yes, the clustermesh-apiserver.
It does not support pprof profiling because it does not import package `"net/http/pprof"`, neither directly nor transitively through `pkg/pprof`, like the agent and the operator do.
i am planning to open a follow-up PR to add support for pprof in the clustermesh-apiserver too.